metadata/namemapping/service_name_mapping.go

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package metadata
+package namemapping
```

Comment:
Could you use one word for this package name

Reply:
maybe namemapping is one word?

Reply:
mapping may be enough.